### PR TITLE
[new release] pp-binary-ints (0.1.0)

### PR DESCRIPTION
**Pull request by ifazk**
Pretty Printing Binary Integers

- Project page: <a href="https://github.com/ifazk/pp-binary-ints">https://github.com/ifazk/pp-binary-ints</a>
- Documentation: <a href="https://ifazk.github.io/pp-binary-ints/">https://ifazk.github.io/pp-binary-ints/</a>

##### CHANGES:

Initial release.

### CODE REVIEW

File: packages/pp-binary-ints/pp-binary-ints.0.1.0/opam [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+maintainer: ["Ifaz Kabir"]
+authors: ["Ifaz Kabir"]
+license: "CC0-1.0"
+tags: ["printf" "format" "boolean" "binary"]
+homepage: "https://github.com/ifazk/pp-binary-ints"
+doc: "https://ifazk.github.io/pp-binary-ints/"
+bug-reports: "https://github.com/ifazk/pp-binary-ints/issues"
+depends: [
+  "dune" {>= "2.9"}
+  "ocaml" {>= "4.10" & < "4.11" | > "4.11" & < "4.14"}
```
Comment (mseri) [inline packages/pp-binary-ints/pp-binary-ints.0.1.0/opam:17]:
```suggestion
  "ocaml" {>= "4.10" & < "4.14"}
```
The way the bounds are written basically does nothing, and the packages was compiled and tested successfully on 4.11 by the CI: https://opam.ci.ocaml.org/github/ocaml/opam-repository/commit/5f209f8c09072673acdd2931e914db2c10f9dd25/variant/opam-2.0,compilers,4.11,pp-binary-ints.0.1.0

File: packages/pp-binary-ints/pp-binary-ints.0.1.0/opam [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+maintainer: ["Ifaz Kabir"]
+authors: ["Ifaz Kabir"]
+license: "CC0-1.0"
+tags: ["printf" "format" "boolean" "binary"]
+homepage: "https://github.com/ifazk/pp-binary-ints"
+doc: "https://ifazk.github.io/pp-binary-ints/"
+bug-reports: "https://github.com/ifazk/pp-binary-ints/issues"
+depends: [
+  "dune" {>= "2.9"}
+  "ocaml" {>= "4.10" & < "4.14"}
```
Comment (kit-ty-kate) [inline packages/pp-binary-ints/pp-binary-ints.0.1.0/opam:17]: What is the reason for this restriction? Why does it not work with 4.14?

Comment (ifazk) [reply packages/pp-binary-ints/pp-binary-ints.0.1.0/opam:17]: I just haven't tested it for 4.14. On second thought, there's actually no good reason for this, do you want me to remove it?

Comment (kit-ty-kate) [reply packages/pp-binary-ints/pp-binary-ints.0.1.0/opam:17]: Yes that would be easier for you (avoiding to do manual changes everytime there is a new release) and us opam-repository maintainers. We check new versions of the compiler against all opam packages before it is release on a regular bases and you will be notified if it ever breaks some point

Comment (ifazk) [reply packages/pp-binary-ints/pp-binary-ints.0.1.0/opam]: Thanks for pushing the change. This is good to know for the future.

**Comment by kit-ty-kate** [comment]
Thanks! Could you return the fix to the opam file upstream so that it can be taken into account for the next release?

**Comment by ifazk** [comment]
Will do. Thanks for helping me getting this merged!

**Pull request merged**